src/foam/mlang/mlang.js

```diff
@@ -1621,7 +1645,7 @@ foam.CLASS({
     },
     {
       name: 'createStatement',
-      javaCode: 'return " ? "; '
+      javaCode: 'return getValue().toString(); '
```

Comment:
I think this needs to be getValue().toSQL or something.  toString will not escape it properly for SQL.

Reply:
Do we need to fix createStatement for this PR?  I'd rather leave that for a separate job, we need to be very careful that we don't introduce SQL injection vulnerabilities.  And I think that's best avoided by not repurposing toString() when we mean toSQL().

Reply:
Oh, is `createStatement` something to do with SQL? I didn't realize that. It's what's being used by MDAO when logging the unoptimized query warning. Should it just be using `toString` there instead of `createStatement`?

Reply:
Reverted changes to `createStatement` and used `toString` instead.

Reply:
Yeah createStatement comes from the foam.dao.SQLStatement interface.  I believe someone was working on PostgreSQL integration, not sure if its finished or not but that's what it's intended for.